components/Select/Select.tsx

```diff
@@ -22,12 +25,15 @@ const Select: FunctionComponent<SelectProps> = props => {
     }),
     menu: provided => ({
       ...provided,
+      color: props.chooseTheme === 'dark' ? '#fff' : '#443e42',
```

Comment:
Add `chooseTheme` to the prop interface. Since we've been using the SelectProps interface from the `react-select` library, you're going to have to create a new interface that extends it, then add that prop to it. Values are specifically either `light` or `dark`, so use a string literal.
Also add defaultProps where u set the default value of `chooseTheme` to `light` ...

Reply:
@edwinmp I extended this interface and added `chooseTheme` with default values. Also added  hover styles for menu items.